tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -111,46 +113,99 @@ type stressConfig struct {
 }
 
 // NewStresser creates stresser from a comma separated list of stresser types.
-func NewStresser(s string, sc *stressConfig, m *member) Stresser {
+func NewStresser(s string, sc *stressConfig, members []*member) Stresser {
```

Comment:
The Implementation I have right now is to have one runner per one type with each runner to be able to talk to all the members. I think several runners of same type can run concurrently, but they shouldn't. the locker racer uses hard code"racers" as its lock. if multiple instances of locker_racer_runners run, they will work but not working in a way that's designed originally.